### ISSUE TITLE
"Monitoring failed" on the mystnodes.com/nodes

### ISSUE THREAD

**Issue by OhLasFar** [mysteriumnetwork/node#5830]
**Describe the bug**
At the moment, after initialising and sending a test. I get "Monitoring failed" on the mystnodes.com/nodes page. Ive shut down one of my nodes and replaced it with the Android Node App. Unsure of if it's a bug OR if it's just a delay. My other node never had this issue 🙂 Also, Just mentionning, ive enabled all 3 in the app, and in the monitoring on the web site, it still shows up as only one is enabled.

**To Reproduce**
I've only done the setup phase and let it sit. (While configuring it correctly of course)
Ive also done this :
1/ force close the node
2/ empty cache (not data, only caches)
3/ launch the app
4/ go to mystnodes.com and refresh until node appears online
5/ on mystnodes.com schedule a node test
6/ see on node UI in the app if you see the test connection from Netherlands

**Expected behavior**
I should be able to send a test to the node AND also the modification made through the app should reflect in the mystnodes.com/nodes portal but after 30 minutes, it still dosent show up as updated. Screen shot will be bellow

**Screenshots**

**Environment (please complete the following information):**
 - Node version: 1.23.1-rc11

 - OS: Android 12, One UI version 4.1, Kernel version 4.14.190-23725627-abG973WVLS7IVK2
 - Device : Samsung S10
 - Identity : 0x6b19cefa6f2f0ceefa469df5f8c761fa978d80ab

**Additional context**
![image](https://github.com/mysteriumnetwork/node/assets/104170187/561046d8-6afb-4481-9784-24841bcaa602)

https://github.com/mysteriumnetwork/node/assets/104170187/d164dd66-8b47-42fb-a009-89acd4d21b8d

**Comment by OhLasFar** [comment]
After ish 1h, the node appears online and is able to be tested. Connection is stable. Unsure of why it took a lot of time, usualy it dosent take that much time. Leaving the ticket open just in case.

**Comment by adinetech** [comment]
Does it mean that now you no longer see Public and B2B VPN & Data transfer services as Disabled?

**Comment by OhLasFar** [comment]
No, so on the app it was enabled for all the 3, but on the dashboard where all my nodes are, only one was there. Took about 1h to update
